### PR TITLE
[marble] no absolute paths

### PR DESCRIPTION
**Pull request by autoantwort**
For #20469

### CODE REVIEW

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Which part of the modification involves the absolute path?

File: ports/marble/portfile.cmake [hunk 1]
```diff
@@ -1,48 +1,46 @@
...
+        -DCMAKE_DISABLE_FIND_PACKAGE_I18n=ON
         -DWITH_KF5=OFF
-)
+)
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake)
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake)
```
Comment (autoantwort) [inline ports/marble/portfile.cmake:20]: This fixed the absolute path

Comment (JackBoosY) [reply ports/marble/portfile.cmake:20]: `vcpkg_fixup_cmake_targets` should do the same job with `vcpkg_cmake_config_fixup`.

**Review by JackBoosY — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**